### PR TITLE
replace text format with merl macro

### PR DESCRIPTION
**Pull request by longlene**

### CODE REVIEW

File: src/erlpool_compile.erl [hunk 1]
```diff
@@ -4,21 +4,28 @@
 
 -export([compile_settings/1]).
 
+-include_lib("syntax_tools/include/merl.hrl").
+
 compile_settings(SettingsList) ->
-    case merl:compile_and_load(get_settings_code(SettingsList)) of
+    Forms = get_settings_code(SettingsList),
+    file:write_file("/tmp/erlpool_globals_gen.erl", erl_prettypr:format(erl_syntax:form_list(Forms))),
```
Comment (silviucpp) [inline src/erlpool_compile.erl]: What is the point of this line ?

**Comment by silviucpp** [comment]
And also I fixed on master a bug when the pool name atom has special chars and need to be escaped. It's now covered by the tests. so please rebase.

File: src/erlpool_compile.erl (hunk 1 above)
Comment (longlene) [reply src/erlpool_compile.erl]: Sorry, it's just for debugging, will comment it later

Comment (silviucpp) [reply src/erlpool_compile.erl]: Thanks. remove that line and also make sure you rebase with my last fix regarding atoms. Then I will merge

Comment (longlene) [reply src/erlpool_compile.erl]: I've merged your master branch, result of common test looks good

**Pull request merged**